dotcom-rendering/src/components/LiveBlogRenderer.tsx

```diff
@@ -62,6 +63,9 @@ export const LiveBlogRenderer = ({
 	selectedTopics,
 	isInLiveblogAdSlotTest = false,
 }: Props) => {
+	const { renderingTarget } = useConfig();
+	const isApps = renderingTarget === 'Apps';
```

Comment:
Rather than rendering everywhere unless it’s `Apps`, should we render only when it’s `Web`?